### ISSUE TITLE
Error: Unrecognized prop: shipping

### ISSUE THREAD

**Issue by spyshower** [thinhvo0108/react-paypal-express-checkout#14]
I just installed the package and attempted to run it with a newly created sandbox app. I got a "Error: Unrecognized prop: shipping" message along with a "ppxo_unhandled_error " one and nothing happens when I click the "Paypal Checkout" button. Each next click of the buttons generates a "ppxo_button_pre_template_click" warning. The web seems to be completely unaware of this.

My code: (no changes from original suggested):

```
import React from 'react';
import PaypalExpressBtn from 'react-paypal-express-checkout';

export default class PaypalCheckout extends React.Component {
    render() {
		const onSuccess = (payment) => {
			// Congratulation, it came here means everything's fine!
            		console.log("The payment was succeeded!", payment);
            		// You can bind the "payment" object's value to your state or props or whatever here, please see below for sample returned data
		}

		const onCancel = (data) => {
			// User pressed "cancel" or close Paypal's popup!
			console.log('The payment was cancelled!', data);
			// You can bind the "data" object's value to your state or props or whatever here, please see below for sample returned data
		}

		const onError = (err) => {
			// The main Paypal's script cannot be loaded or somethings block the loading of that script!
			console.log("Error!", err);
			// Because the Paypal's main script is loaded asynchronously from "https://www.paypalobjects.com/api/checkout.js"
			// => sometimes it may take about 0.5 second for everything to get set, or for the button to appear
		}

    
		let env = 'sandbox'; // you can set here to 'production' for production
		let currency = 'EUR'; // or you can set this value from your props or state
		let total = 1; // same as above, this is the total amount (based on currency) to be paid by using Paypal express checkout
		// Document on Paypal's currency code: https://developer.paypal.com/docs/classic/api/currency_codes/

		const client = {
			sandbox:    'xxxx'
			production: 'YOUR-PRODUCTION-APP-ID',
		}
		// In order to get production's app-ID, you will have to send your app to Paypal for approval first
		// For sandbox app-ID (after logging into your developer account, please locate the "REST API apps" section, click "Create App"):
		//   => https://developer.paypal.com/docs/classic/lifecycle/sb_credentials/
		// For production app-ID:
		//   => https://developer.paypal.com/docs/classic/lifecycle/goingLive/

		// NB. You can also have many Paypal express checkout buttons on page, just pass in the correct amount and they will work!
        return (
            <PaypalExpressBtn env={env} client={client} currency={currency} total={total} onError={onError} onSuccess={onSuccess} onCancel={onCancel} />
        );
    }
}

```

Errors:

<img width="1271" alt="screen shot 2017-11-11 at 21 49 02" src="https://user-images.githubusercontent.com/7046283/32692950-5ab0ad9e-c72a-11e7-9fdf-f159230c46e5.png">
<img width="1270" alt="screen shot 2017-11-11 at 21 48 46" src="https://user-images.githubusercontent.com/7046283/32692951-5ad9376e-c72a-11e7-9b83-2fc2ff08d950.png">

**Comment by spyshower** [comment]
I just saw the other identical issue... Strange no search brought it up.

**Comment by spyshower** [comment]
Downgrading to version 1.01 works but I had to make a few changes to the prop types because of an error: include prop-types and modify the 
`ScriptLoader.propTypes`value to `onScriptLoaded: PropTypes.func`

**Comment by yashwanthm** [comment]
@spyshower I have a similar issue with propTypes using this package. I'm using WebPack. Can you help me out with where exactly this change needs to be made?

**Comment by spyshower** [comment]
@yashwanthm go to `your_project_folder/node_modules/react-paypal-express-checkout/node_modules/react-async-script-loader/lib/index.js`:

insert `var PropTypes = require( 'prop-types');` (i think it's not there by default)

and at line 158 or so:
```
ScriptLoader.propTypes = {
      onScriptLoaded: PropTypes.func
};
ScriptLoader.defaultProps = {
      onScriptLoaded: _utils.noop
};
```

**Comment by rashmimhatre100** [comment]
props.js:18 Uncaught Error: Unrecognized prop: shipping Same issue

**Comment by MieszkoWrzeszczynski** [comment]
I also have the same problem.

**Comment by heyhujiao** [comment]
I also have the same problem

**Comment by macsplan** [comment]
me too

**Comment by thinhvo0108** [comment]
I have just updated the latest version (@1.0.4)
Could you guys please try using it? Thanks!

**Comment by thinhvo0108** [comment]
Close this issue, which was probably fixed in version @1.0.4
